idea/src/META-INF/plugin.xml

```diff
@@ -1839,6 +1844,14 @@
                      language="kotlin"
     />
 
+    <localInspection implementationClass="org.jetbrains.kotlin.idea.intentions.RemoveEmptyClassBodyInspection"
+                     displayName="Replace empty class body"
+                     groupName="Kotlin"
+                     enabledByDefault="true"
+                     level="WARNING"
```

Comment:
There is nothing bad about an empty class body, so this should be severity INFO, not WARNING